pkg/kubernetes/scripts/kube-client-cockpit.js

```diff
@@ -1052,8 +1052,14 @@
                     },
                     flavor: "kubernetes",
                     isAdmin: false,
+                    currUser: null,
                     canChangeConnection: false,
                 };
+                var curr_user = loader.load("/oapi/v1/users/~")
+                    .then(function(response){
+                        if (response)
+                            settings.currUser = response;
+                    }, function(ex) {});
```

Comment:
We should only allow 404 to fall through here. Other failures should result in a real failure displayed in the UI.